config/examples/library.config

```diff
@@ -0,0 +1,12 @@
+ARCH=
```

Comment:
Trying to compile with this configuration does not create a `target.h` file. I see there is a procedure described in lib.md. Would it be possible to have all the steps in a Makefile target? Example:

```
test-lib: include/target.h $(OBJS)
       $(Q)$(CC) $(CFLAGS) -o $@ $^
       touch empty
       $(SIGN_TOOL) --ed25519 --sha256 empty ed25519.der 1
       ./$(@) empty_v1_signed.bin`
```
And adding the following lines to `config/examples/library.c`:

 ```
+WOLFBOOT_SECTOR_SIZE=0x20000
+WOLFBOOT_PARTITION_SIZE=0x20000
```

Reply:
ok I put a solution in place. I'll probably update the `lib.md` and squash a few commits.